src/client/components/TripFriend/TripFriend.jsx

```diff
@@ -4,7 +4,9 @@ import Img from 'react-image';
 import Spinner from '../../components/Spinner';
 
 const TripFriend = (props) => {
-  const { tripFriend, isAdminShown } = props;
+  const { tripFriend, isAdminShown, anonCounter } = props;
+
+
   return (
     <div className="trip-friend">
       { tripFriend.profile_pic
```

Comment:
just replace all `tripFriend.profile_pic` to `tripFriend.profile_photo` to show user's profile photo

Reply:
good catch, thanks!

Reply:
done